db/migrate/20160829134746_remove_atomic_container_providers.rb

```diff
@@ -0,0 +1,41 @@
+    ManageIQ::Providers::Atomic::ContainerManager::MetricsCollectorWorker::Runner ManageIQ::Providers::Openshift::ContainerManager::MetricsCollectorWorker::Runner
+    ManageIQ::Providers::Atomic::ContainerManager::RefreshParser                  ManageIQ::Providers::Openshift::ContainerManager::RefreshParser
+    ManageIQ::Providers::Atomic::ContainerManager::RefreshWorker                  ManageIQ::Providers::Openshift::ContainerManager::RefreshWorker
+    ManageIQ::Providers::Atomic::ContainerManager::RefreshWorker::Runner          ManageIQ::Providers::Openshift::ContainerManager::RefreshWorker::Runner
+    ManageIQ::Providers::Atomic::ContainerManager::Refresher                      ManageIQ::Providers::Openshift::ContainerManager::Refresher
+
+    ManageIQ::Providers::AtomicEnterprise::ContainerManager                                 ManageIQ::Providers::OpenshiftEnterprise::ContainerManager
+    ManageIQ::Providers::AtomicEnterprise::ContainerManager::EventCatcher                   ManageIQ::Providers::OpenshiftEnterprise::ContainerManager::EventCatcher
+    ManageIQ::Providers::AtomicEnterprise::ContainerManager::EventCatcher::Runner           ManageIQ::Providers::OpenshiftEnterprise::ContainerManager::EventCatcher::Runner
+    ManageIQ::Providers::AtomicEnterprise::ContainerManager::EventParser                    ManageIQ::Providers::OpenshiftEnterprise::ContainerManager::EventParser
+    ManageIQ::Providers::AtomicEnterprise::ContainerManager::MetricsCollectorWorker         ManageIQ::Providers::OpenshiftEnterprise::ContainerManager::MetricsCollectorWorker
+    ManageIQ::Providers::AtomicEnterprise::ContainerManager::MetricsCollectorWorker::Runner ManageIQ::Providers::OpenshiftEnterprise::ContainerManager::MetricsCollectorWorker::Runner
+    ManageIQ::Providers::AtomicEnterprise::ContainerManager::RefreshParser                  ManageIQ::Providers::OpenshiftEnterprise::ContainerManager::RefreshParser
+    ManageIQ::Providers::AtomicEnterprise::ContainerManager::RefreshWorker                  ManageIQ::Providers::OpenshiftEnterprise::ContainerManager::RefreshWorker
+    ManageIQ::Providers::AtomicEnterprise::ContainerManager::RefreshWorker::Runner          ManageIQ::Providers::OpenshiftEnterprise::ContainerManager::RefreshWorker::Runner
+    ManageIQ::Providers::AtomicEnterprise::ContainerManager::Refresher                      ManageIQ::Providers::OpenshiftEnterprise::ContainerManager::Refresher
+  )]
+
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class Authentications < ActiveRecord::Base; end
+
+  def up
+    say_with_time "Rename class references for Atomic and AtomicEnterprise" do
+      rename_class_references(NAME_MAP)
+    end
+
+    Authentications.update_all("name = replace(name, 'ManageIQ::Providers::Atomic', 'ManageIQ::Providers::Openshift')")
```

Comment:
@blomquisg it seems strange that the authentication name contains the provider type as part of a string and not in a separate column. Authentication even has resource_type that currently holds ExtManagementSystem

e.g 
```
name | ManageIQ::Providers::Openshift::ContainerManager Molecule
```